app/components/PageTitle.js

```diff
@@ -1,15 +1,20 @@
 // @flow
 import * as React from "react";
 import { Helmet } from "react-helmet";
+import AuthStore from "stores/AuthStore";
 
 type Props = {
   title: string,
   favicon?: string,
+  auth: AuthStore,
 };
 
+const { auth } = this.props;
+const { team } = auth;
+
 const PageTitle = ({ title, favicon }: Props) => (
   <Helmet>
-    <title>{`${title} - Outline`}</title>
+    <title>{`${title} - ${team.name}`}</title>
```

Comment:
Similarly team might be empty again here and the product name should remain at the end of the title.